lib/linguist/languages.yml

```diff
@@ -1458,6 +1458,16 @@ Eagle:
   codemirror_mode: xml
   codemirror_mime_type: text/xml
   language_id: 97
+Earthly:
+  type: programming
+  extensions:
+  - ".earthfile"
```

Comment:
Please remove this... there doesn't appear to be a single `.earthfile` on GitHub so it's a very very long way from meeting our minimum usage requirement 😁